frontend/src/components/navbar.js

```diff
@@ -1,57 +1,73 @@
-              <Nav className='mr-auto' navbar tabs>
-              <NavItem>
-                  <NavLink href="/playlist">PLAYLIST</NavLink>
-                </NavItem>
-                <NavItem>
-                  <NavLink href="/user">USER</NavLink>
-                </NavItem>
-                <NavItem>
-                  <NavLink href="/aboutus">ABOUT</NavLink>
-                </NavItem>
-                <NavItem>
-                  <NavLink href="http://localhost:8888">LOGIN</NavLink>
-                </NavItem>
-              </Nav>
-              <Form inline>
-                <InputGroup>
-                    <Input placeholder = 'keyword'></Input>
-                </InputGroup>
-                <Button variant='outline-success'>Search</Button>
-              </Form>        
-            </Navbar>
-          </div>
-          <Switch>
+    constructor(props) {
+        super(props);
+        this.state = {
+            current: 'mail',
+            };
+    }
 
-          <Route exact path = '/' component = {homePage} />
+    handleClick = e => {
+        console.log('click ', e);
+        this.setState({ current: e.key });
+    };
 
+  render() {
+    const { current } = this.state;
+    return (
+        <Router>
+            <div className="navbarContainer">
+                <Menu onClick={this.handleClick} selectedKeys={[current]} mode="horizontal">
+                    <Menu.Item id="home" key="home" icon={<HomeOutlined />}>
+                        <a href="/">Playlist Creator</a>
+                    </Menu.Item>
+                    <Menu.Item key="user" icon={<UserOutlined />}>
+                        <a href="/user">User</a>
+                    </Menu.Item>
+                    <SubMenu id="dropDown" icon={<AppstoreAddOutlined />} title="Create Playlist">
+                        <Menu.Item key="concert" icon={<PlusOutlined />}>
+                            <a href="/playlist">Concert Playlist</a>
+                        </Menu.Item>
+                        <Menu.Item key="normal" icon={<PlusOutlined />}>
+                            <a href="/playlist">Normal Playlist</a>
+                        </Menu.Item>
+                    </SubMenu>
+                    <Menu.Item key="aboutUs" icon={<QuestionOutlined />}>
+                        <a href="/aboutUs">About Us</a>
+                    </Menu.Item>
+                    <Menu.Item key="github" icon={<GithubOutlined />}>
+                        <a href="https://github.com/csjoblinksreddit/playlistapplication">Github</a>
+                    </Menu.Item>
+                    <Menu.Item id="login" key="login" icon={<LoginOutlined />}>
+                        <a href="http://localhost:8888">Login</a>
+                    </Menu.Item>
+                </Menu>
+            </div>
+            <Switch>
+              <Route exact path = '/' component = {homePage} />
+              <Route exact path = '/concertPlaylist' component = {concertPlaylist} />
```

Comment:
We have to remove this as well, if not we get an error saying that it cannot find concertPlaylist

Reply:
Done.